services/src/main/java/org/fao/geonet/services/metadata/format/Format.java

```diff
@@ -214,6 +217,12 @@ public void execXml(
             metadata = getXmlFromUrl(context, lang, url, request);
         }
         Element metadataEl = Xml.loadString(metadata, false);
+
+        if(mdPath != null) {
+            final List<Namespace> namespaces = context.getBean(SchemaManager.class).getSchema(schema).getNamespaces();
+            metadataEl = Xml.selectElement(metadataEl, mdPath, namespaces);
```

Comment:
Actually, the getSchema method throws an exception if the schema does not exist
`Schema not registered : iso1913d9`
So there is no NPE